### DISCUSSION TITLE
Unable to get font files in an Astro project

### DISCUSSION THREAD

**Discussion by ymcheung** [vercel/community#6388] [Help]
### Summary

Hello!

I want to create dynamic open graph images by satori in an Astro project and deploy it to Vercel (hobby plan).

When I called the astro api route in local development env, it works fine.

However, it always returned error 500 in preview deployments.

### Example

The code is in a Github private repo actually.

https://codesandbox.io/p/devbox/pensive-swirles-8r5d75?file=%2Fsrc%2Fpages%2Fapi%2F%5BpostId%5D.png.ts%3A34%2C19&workspaceId=d1419148-7ecd-4e85-9069-e54b7e137fd9

### Steps to Reproduce

Calling the API route with GET method in preview deployments always returned 500 error and the logs tab read:

```
Error: ENOENT: no such file or directory, open '/var/task/public/fonts/nunito-sans-latin-500-normal.ttf'
  at Object.openSync (node:fs:596:3)
  ...
```

I tried
```
const file = readFileSync(path.resolve('./public/fonts/nunito-sans-latin-500-normal.ttf'))
```
or

```
const filePath = path.join(
  process.cwd(),
  'public'
  'fonts',
  'nunito-sans-latin-500-normal.ttf'
  );
const fontFile = readFileSync(filePath);
```
Both of them caused the error.

Did I miss something to setup? Thank you!

**Comment by versecafe** [comment]
@ymcheung It's likely that when being bundled up for running the function on the lambda the file is not included, also you shouldn't need to read the fontFile from FS, either import the font directly or call on the font from `YOURURL/path/to/font` in this case `yoururl.com/fonts/nunito-sans-latin-500-normal.ttf`
